src/conn/conn_layered.c

```diff
@@ -1301,12 +1303,23 @@ __wti_disagg_conn_config(WT_SESSION_IMPL *session, const char **cfg, bool reconf
         WT_STAT_CONN_SET(session, disagg_role_leader, leader ? 1 : 0);
     } else if (!was_leader && leader) {
         /* Follower step-up. */
+        time_start = __wt_clock(session);
         WT_WITH_CHECKPOINT_LOCK(session, ret = __disagg_step_up(session));
+        time_stop = __wt_clock(session);
         WT_ERR_MSG_CHK(session, ret, "Failed to step up to the leader role");
-    } else if (was_leader && !leader)
+        WT_STAT_CONN_SET(session, disagg_step_up_time, WT_CLOCKDIFF_MS(time_stop, time_start));
+        __wt_verbose_debug1(session, WT_VERB_DISAGGREGATED_STORAGE,
+          "Step up completed in %" PRIu64 " milliseconds", WT_CLOCKDIFF_MS(time_stop, time_start));
+    } else if (was_leader && !leader) {
+        time_start = __wt_clock(session);
```

Comment:
OCD nit:

I'd make the sequence here match the one for step-up
```
   /* comment */
   time_start = ...
   WT_WITH_CHECKPOINT_LOCK(...)
   time_stop = ...
```

I.e., swap the comment with the line setting `time_start`